shared/whats-new/icon/index.tsx

```diff
@@ -6,112 +6,138 @@ import {keybaseFM} from '../../constants/whats-new'
 import Popup from '../popup.desktop'
 
 type Props = {
+  newRelease: boolean
+  isProfileHeader: boolean // desktop only
   color?: string
-  badgeColor?: string
-  style?: IconStyle
   className?: string
-  newRelease: boolean
+  style?: IconStyle
   onClick?: () => void
 }
 
 type PopupProps = Props & {
+  // Desktop only
+  updateAvailable: boolean
   // Desktop only
   attachToRef: React.RefObject<Kb.Box2>
 }
 
+// Clips the rainbow background around the icon
 const realCSS = `
```

Comment:
i believe you can actually just use a regular style for this

Reply:
@chrisnojima the style key `WebKitBackgroundClip` is ignored. We might have bumped versions since I wrote that, so I can try again.